src/Fieldtypes/Date.php

```diff
@@ -131,6 +131,17 @@ public function process($data)
 
     public function preProcessIndex($data)
     {
+        if (! $data) {
+            return;
+        }
+
+        if ($this->config('mode') === 'range') {
+            $start = Carbon::parse($data['start'])->format(config('statamic.cp.date_format'));
+            $end = Carbon::parse($data['end'])->format(config('statamic.cp.date_format'));
+
+            return $start.' to '.$end;
```

Comment:
Can you please change `to` to a `–`?

That way we don't have to deal with a translation (since `to` is English). A dash is fine for a range.

I'd make this change myself but it seems I don't have permission.

Reply:
Absolutely, that' a good call. 👍